### PR TITLE
Fix jassimp import of light intensities

### PR DESCRIPTION
**Pull request by NolaDonato**
GearVRF assumes these are 0-1, in jassimp they can be 0 to anything. Jassimp now scales the light intensity  by the maximum R,G,B so the light intensity won't be > 1.
This should improve the look of assets which use lights with very large intensities like the models in the VRBenchmark app.

GearVRF DCO signed off by: Nola Donato nola.donato@samsung.com

### CODE REVIEW

File: GVRf/Framework/framework/src/main/java/org/gearvrf/GVRJassimpAdapter.java [hunk 1]
```diff
@@ -626,17 +635,21 @@ private void importLights(List<AiLight> lights, Hashtable<String, GVRLightBase>
     }
 
     private void setLightProp(GVRLightBase gvrLight, AiLight assimpLight){
-        gvrLight.setFloat("attenuation_constant", assimpLight.getAttenuationConstant());
-        gvrLight.setFloat("attenuation_linear", assimpLight.getAttenuationLinear());
-        gvrLight.setFloat("attenuation_quadratic", assimpLight.getAttenuationQuadratic());
+        //gvrLight.setFloat("attenuation_constant", assimpLight.getAttenuationConstant());
```
Comment (liaxim) [inline GVRf/Framework/framework/src/main/java/org/gearvrf/GVRJassimpAdapter.java]: If those are truly not needed anymore why not remove the method altogether?

Comment (roshanch) [inline GVRf/Framework/framework/src/main/java/org/gearvrf/GVRJassimpAdapter.java]: where do we set these 3 constants from assimp?

**Comment by liaxim** [comment]
Demos run fine. Code looks good except that for lights imported from assimp the attenuation_constant, attenuation_linear and attenuation_quadratic do not seem to be set anymore. Maybe this is intended but it is unclear from the code.

@roshanch any suggestions for models with lights we can test?

**Comment by liaxim** [comment]
I also rebased the pull request. There was a merge conflict in GVRTexture.

**Comment by roshanch** [comment]
@liaxim we have the cube models with lights in it. but i doubt it will be good test for the attenuation. Papiya has reported the issue with benchmark app with problem with light intensity for direct light, she will test this change with that app.

File: GVRf/Framework/framework/src/main/java/org/gearvrf/GVRJassimpAdapter.java (hunk 1 above)
Comment (liaxim) [inline GVRf/Framework/framework/src/main/java/org/gearvrf/GVRJassimpAdapter.java]: jassimp.cpp:loadLights()?

**Comment by liaxim** [comment]
Ok then. If you are ok with merging this then I am ok too.

**Comment by roshanch** [comment]
let me talk with Papiya about VRBenchmark app Nola has mentioned. I will check whether it works with that

File: GVRf/Framework/framework/src/main/java/org/gearvrf/GVRJassimpAdapter.java (hunk 1 above)
Comment (NolaDonato) [inline GVRf/Framework/framework/src/main/java/org/gearvrf/GVRJassimpAdapter.java]: I screwed up and commented out the attenutation settings. Put them back in and you are good to go.

**Comment by liaxim** [comment]
Per Nola, I uncommented the attenuation settings.

**Comment by thomasflynn** [comment]
so, what's the latest status on this?

**Comment by liaxim** [comment]
We can merge it.

**Pull request merged**